src/coreclr/vm/jitinterface.cpp

```diff
@@ -7497,12 +7515,8 @@ static void getMethodInfoHelper(
     // Having a header means there is backing IL for the method.
     if (NULL != cxt.Header)
     {
-        scopeHnd = cxt.HasTransientMethodDetails()
```

Comment:
I am not able to tell what this tried to do before this PR. Can `HasTransientMethodDetails` ever be true here before this PR?

Reply:
I don't believe so. I think this was me being defensive at the time.